autocertkit/network_tests.py

```diff
@@ -1002,6 +1002,85 @@ def test_ping(self, session):
         return self._run_test(session)
 
 
+class MulticastTest(IperfTest):
+    """ Subclass that adds multicast specific operations"""
+    MULTICAST_IP = '226.94.1.1'
```

Comment:
@bpwook @vivekkumac  The full range of multicast addresses is from 224.0.0.0 to 239.255.255.255. I just pick one of them.
I will move this line to util.py